wsdl/ver10/schema/onvif.xsd

```diff
@@ -1807,12 +1807,106 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 							<xs:documentation>Volume setting of the output. The applicable range is defined via the option AudioOutputOptions.OutputLevelRange.</xs:documentation>
 						</xs:annotation>
 					</xs:element>
+					<xs:element name="EQPresetToken" type="tt:ReferenceToken" minOccurs="0">
+						<xs:annotation>
+							<xs:documentation>Optional audio EQ preset token to be applied to the audio output.</xs:documentation>
+						</xs:annotation>
+					</xs:element>
 					<xs:any namespace="##any" processContents="lax" minOccurs="0" maxOccurs="unbounded"/>   <!-- first Vendor then ONVIF -->
 				</xs:sequence>
 				<xs:anyAttribute processContents="lax"/>
 			</xs:extension>
 		</xs:complexContent>
 	</xs:complexType>
+
+	<xs:simpleType name="FrequencyRange">
+    <xs:restriction base="xs:string">
```

Comment:
is this frequency range list standard for all type of EQPresets?

Reply:
> is this frequency range list standard for all type of EQPresets?

This is the common frequency range list, it can be different also. This enumeration is only for reference, the actual data type for this frequency range is xs:string, therefore we can extend it in future.

Reply:
I like the xs:string for further extensibility.